runcommandonset/src/runcommand.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Clone, PartialEq, Serialize)]
+pub struct RunCommand {
+    pub executable: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub arguments: Option<Vec<String>>,
+    // default value for exit code is 0
+    #[serde(default, skip_serializing_if = "is_default")]
```

Comment:
Does this mean it doesn't serialize `exit_code` if the value is 0?  Also, for serialization, this should be renamed to `exitCode` to align with JSON naming convention.

Reply:
Yes - I did that so `exit_code` was not required as an input but would still be initialized internally to `0`, but might make more sense to accomplish that with `Option` depending on user feedback?

If `exit_code` isn't provided, I see no need to write it back out for a `get` command, but I could see it being useful for a `set` command. 

As is, for `set` the `changedProperties` will capture any difference in `exit_code`